mpas_analysis/config.default

```diff
@@ -299,14 +299,73 @@ baseDirectory = /dir/to/seaice/reference
 [timeSeriesOHC]
 ## options related to plotting time series of ocean heat content (OHC)
 
+# plot S, T, and OHC fields themselves, in addition to their anomalies?
+plotOriginalFields = False
 # compare to observations?
 compareWithObservations = False
 # list of region indices to plot from the region list in [regions] below
 regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
+# Number of points over which to compute moving average for time series (e.g., 
+#for monthly output, movingAveragePoints=12 corresponds to a 12-month moving
+# average window)
 movingAveragePoints = 12
```

Comment:
It would be best to rename this to `movingAveragePointsTimeSeries` to be consistent with `movingAveragePointsHovmoller` below.